content/blog/asyncapi_codegen_scst.md

```diff
@@ -0,0 +1,186 @@
+---
+type: blog
+featured: true
+cover: /images/posts/asyncapi-codegen_pic-00.webp
+weight: 100
+title: "AsyncAPI Code Generation: Microservices Using Spring Cloud Stream"
+date: 2020-04-28 17:39:34 UTC
+tags:
+   - solace
+   - asyncapi
+   - java
+   - spring
+   - EDA
+authors:
+   - name: Marc DiPasquale
+     photo: /images/avatars/mdipasquale.webp
+     link: https://twitter.com/mrc0113
+     byline: Developer Advocate at Solace
+---
+
+Code generation is no simple feat. There are a lot of complexities when it comes to generating useful application code. In this post, I am going to walk you through generating your own microservices using Spring Cloud Stream and the AsyncAPI Code Generator. These tools should help to simplify things when defining and implementing your asynchronous applications. I explained the same idea in a video you can [watch here](https://www.youtube.com/watch?v=QEDL6AqsaJc), and all of the artifacts are [available in GitHub](https://github.com/Mrc0113/asyncapi-codegen-scst). This post [AsyncAPI Code Generation: Microservices Using Spring Cloud Stream](https://solace.com/blog/asyncapi-codegen-microservices-using-spring-cloud-stream/) appeared first on [Solace](https://solace.com).
+
+# **AsyncAPI: What Is It?** 
```

Comment:
please do not bold headings as it doesn't affect styling at all, and we should anyway not manupulate styling of headings through markdown

Reply:
you have it in few places

Reply:
fixed in next commit